Specs/DateTime/French/HolidayExtractor.json

```diff
@@ -141,5 +141,15 @@
         "Length": 14
       }
     ]
-  }
-]
\ No newline at end of file
+  },
+  {
+      "Input": "J'y retournerai à Noël",
+      "Results": [
+        {
+          "Text": "Noël",
+          "Type": "date",
+          "Start": 16,
+          "Length": 9
+       }
+     ]
+   },
```

Comment:
There should be no "," here and the next line needs to finish the spec with ]. Like in the lines in red above.
Other than this, it seems you got the process.

Reply:
The build log shows where the error is:
WARNING: Invalid array passed in, extra trailing ','. (2906): [